1223-curator_worklist/1375-curator_views_doi_request_tab_part_ii.feature

```diff
@@ -0,0 +1,12 @@
+    @1375
+    Scenario: Curator views DOI Request tab (Part II)
+        Given that the user opens My Worklist
+        When they click the tab:
+            | DOI request |
+        Then they see a list with expandable items with fields:
```

Comment:
I assume that not all messages will be shown before the expandable item is expanded. What is shown before the expansion and what after the expansion?

Reply:
NP-531 is before expansion and NP-1252 is after.